src/main/java/org/jenkinsci/plugins/awsdevicefarm/AWSDeviceFarmRecorder.java

```diff
@@ -329,9 +329,9 @@ public AWSDeviceFarmRecorder(String projectName,
 
         if (ifWebApp != null && ifWebApp) this.ifWebApp = true;
         else this.ifWebApp = false;
-        if (ifWebApp && testToRun.equalsIgnoreCase("APPIUM_PYTHON")) this.testToRun = "APPIUM_WEB_PYTHON";
-        else if (ifWebApp && testToRun.equalsIgnoreCase("APPIUM_JAVA_JUNIT")) this.testToRun = "APPIUM_WEB_JAVA_JUNIT";
-        else if (ifWebApp && testToRun.equalsIgnoreCase("APPIUM_JAVA_TESTNG")) this.testToRun = "APPIUM_WEB_JAVA_TESTNG";
+        if (this.ifWebApp && testToRun.equalsIgnoreCase("APPIUM_PYTHON")) this.testToRun = "APPIUM_WEB_PYTHON";
```

Comment:
Why are you doing what appears to be duplicate logic here and in `isTestType`?

You should really refactor this blob out into a method because this code is not easy to read. Please use `{}` as well.